source/Calamari.Tests/Helpers/TestEnvironmentExtended.cs

```diff
@@ -0,0 +1,35 @@
+using Assent;
+using Assent.Namers;
+using Calamari.Testing.Helpers;
+
+namespace Calamari.Tests.Helpers
+{
+    public static class TestEnvironmentExtended
```

Comment:
A class better name would be `AssentConfigurations`, with the properties renamed to `Default`, `Json`, `Yaml`, etc.

Reply:
Good idea, done.